addons/web/static/src/js/owl_compatibility.js

```diff
@@ -392,8 +392,11 @@ odoo.define('web.OwlCompatibility', function () {
          */
         destroy() {
             if (this.parentWidget) {
-                const index = children.get(this.parentWidget).indexOf(this);
-                children.get(this.parentWidget).splice(index, 1);
+                const parentChildren = children.get(this.parentWidget);
```

Comment:
So parentWidget may or may not have been added to children ?
or it may or may not already have been deleted from children ?

I suppose this change is only preemptive though

Reply:
We definitely had the bug at some point if we introduced this fix. Since then it may or may not have been fixed in the past months.

I don't remember the exact case cc @alexkuhn

Reply:
We had a crash without this "fix", haven't checked recently if it still applies, but likely so because chatter container is mounted on fragment due to `form_renderer:renderView()` implementation details.